maquetado/detail.html

```diff
@@ -11,23 +11,23 @@
 <body>
   <div class='back-container'>
     <a class='back-arrow' href='index.html'>Atrás</a>
-    <div class='book-card'>
-      <div class='book-cover'>
-        <img class='book-image' src='assets/book-cover.png'>
-        <img class='book-badge' src='assets/badge.png'>
-      </div>
-      <div class='book'>
-        <div class='book-title'>
-          <h1 class='title'>Título del Libro</h1>
-          <p class='title-genre'>(género)</p>
-          <hr class='book-separator'>
-        </div>
-        <div class='book-details'>
-          <p class='book-item'><strong>Autor del Libro: </strong> <span class='book-item-span'>del autor del Libro</span></p>
-          <p class='book-item'><strong>Editorial: </strong><span class='book-item-span'>Nombre de editorial</span></p>
-          <p class='book-item'><strong>Año de Publicación: </strong><span class='book-item-span'>Año de Publicación</span></p>
-        </div>
-      </div>
+  </div>
+  <div class='book-card'>
+    <div class='book-cover'>
+      <img class='book-image' src='assets/book-cover.png'>
+      <img class='book-badge' src='assets/badge.png'>
+    </div>
+    <div class='book-title'>
+      <h1 class='title'>Título del Libro</h1>
+    </div>
+    <div class='title-genre'>
+      <p class='title-genre'>(género)</pclass='title-genre'>
+    </div>
+    <hr class='book-separator'>
```

Comment:
It is not necessary, you can move the book-separator attributes to `p` for example

Reply:
Oka! I moved it to book-detail and delete that separator.